Docs/feature_memory_protection.md

```diff
@@ -401,3 +401,66 @@ manner similar to below:
 ```
 
 before building the HOB.
+
+## **Memory Protection Special Regions**
+
+Memory protection is not activated until the CPU Architecture Protocol has been installed because the
+protocol facilitates access to the attribute set functions in CpuDxe. Many allocations and image loads
+will have occurred by the time the protocol is published so careful accounting is required to ensure
+appropriate attributes are applied. An event notification triggered on the CPU Architecture Protocol
+installation will combine the GCD and EFI memory maps to create a full map of memory. Because image memory

```

Comment:
"full map of memory"

Can you add that this full map of memory is internal to the memory protection code, and that it is not reflected in the GCD/EFI Memory map?

Reply:
Updated